### ISSUE TITLE
Add Keyboard Shortcut to copy file names

### ISSUE THREAD

**Issue by pota2dev** [bradzacher/vscode-copy-filename#1]
Add a keyboard shortcut to copy file names when file(s) are selected in File Explorer. If the editor is on focus, copy filename of current file instead.

**Comment by Melebius** [comment]
Please, make keyboard shortcuts possible. I tried to set keyboard shortcuts for `Copy Filename (Without Extension)` in VS Code, but they don’t work.

**Comment by Victor-IX** [comment]
You can already add keyboard shortcuts to those copy functions in the Keyboard Shortcuts Editor:

<img width="1987" height="298" alt="Image" src="https://github.com/user-attachments/assets/9824ed0b-7e62-4f9d-8321-2b2a9f8344cf" />

**Comment by bradzacher** [comment]
yup -- you should be able to add keyboard shortcuts -- just search for `copyFilename` in the keyboard shortcut settings window as Victor showed!
